src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -156,6 +207,74 @@ private bool IsIfNullCheck(IOperation statement, IParameterSymbol parameter)
             return false;
         }
 
+        protected IBlockOperation GetBlockStatmentOpt(Document document, SemanticModel semanticModel, SyntaxNode functionDeclaration, CancellationToken cancellationToken)
```

Comment:
Logic in this method is mostly copied from `AbstractInitializeParameterCodeRefactoringProvider`, please expose a proper helper in the base type instead of duplicate code.

Reply:
True. Got rid of it.